File: Python/helpaphd.py
'''
 *
 *	Problem: "Help a PhD candidate out!"
 *
'''

'''-----main-----'''
cases = int(input())

for i in range(cases):
	case = input()
	
	if "+" in case:
		operands = case.split("+")
		a, b = int(operands[0]), int(operands[1])
		result = a + b
		print(result)
	elif "=" in case:
		print("skipped")

File: Java/helpaphd.java
/*
 *	Problem: "Help a PhD candidate out!"
 */

import java.util.Scanner;

public class helpaphd {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int cases = sc.nextInt();
		
		for (int i = 0; i < cases; i++ ) {
			String line = sc.next();
			if (line.contains("+")) {
				String[] operands = line.split("\\+");
				int a = Integer.parseInt(operands[0]);
				int b = Integer.parseInt(operands[1]);
				int result = a + b;
				System.out.println(result);
			}
			else if (line.contains("=")) {
				System.out.println("skipped");
			}
		}
	}
	
}